selenium/src/test/java/au/org/emii/portal/PortalUtil.java

```diff
@@ -0,0 +1,17 @@
+package au.org.emii.portal;
```

Comment:
@jonescc  Do you think we can move this under package _au.org.emii.portal.utils_